robocorp-code/src/robocorp_code/inspector/windows/robocorp_windows/__init__.py

```diff
@@ -120,6 +120,7 @@ def find_window(
     timeout: Optional[float] = None,
     wait_time: Optional[float] = None,
     foreground: bool = True,
+    skip_cursor_centering: bool = False,
```

Comment:
Having a negation form is usually a bit strange and usually it's nicer to have the argument in a positive form.

i.e.:
```
move_cursor_to_center: bool = True
```

Reply:
OK. Let me make the change.